homework.md

```diff
@@ -44,67 +44,70 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is the Programming Language for the Web
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+- It helps people to share code/information,
+- It used to keep the history of changes, 
+- It makes collaboration easier.
```

Comment:
```suggestion
- It helps people to share code/information,
- It used to keep the history of changes, 
- It makes collaboration easier.
```
This is great - well done! One thing you can improve on is whether there are any other benefits of Git/GitHub. The first and last points are very similar. Great job!

Reply:
Thank you very much